tests/aiohttp/test_graphql_transport_ws.py

```diff
@@ -879,7 +879,7 @@ async def test_single_result_operation_error(aiohttp_client):
         assert ws.closed
 
 
-async def test_single_result_operation_exception(aiohttp_client):
+async def test_single_result_operation_exception(aiohttp_client) -> None:
```

Comment:
can we skip this rule for tests? I don't think it adds any value :)

Reply:
right on, will fix that